### PR TITLE
[OperationalInsights] Support for new commands and update existing 

### PR DESCRIPTION
**Pull request by dabenhamMic**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
I notice you add new cmdlets in your PR and so please submit a cmdlet review here https://github.com/Azure/azure-powershell-cmdlet-review-pr

Before submit the review, please refer to the [cmdlet design guideline] (https://github.com/Azure/azure-powershell/tree/master/documentation/development-docs/design-guidelines) first.

In static analysis check, break changes are detected in your codes. Please refer to https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md and avoid breaking change. We can discuss in cmdlet review in details.

You may want to add some tests or record existing test cases. Please refer to the https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/azure-powershell-developer-guide.md for more details.

**Comment by dabenhamMic** [comment]
@msJinLei added new cmdlets request issue:  https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/951

**Comment by dabenhamMic** [comment]
Hi @msJinLei,
@yoramsinger asked me to stop updating and expanding OperationalInsights PSH module. 
I have to utilize the existing commands and update their contracts.
what is needed to approve and merge this PR?

**Comment by BethanyZhou** [comment]
Hi @dabenhamMic , some revisions need be done in cmdlet design doc, please have a look.

**Comment by dabenhamMic** [comment]
@BethanyZhou what else is missing here?

**Comment by BethanyZhou** [comment]
@dabenhamMic , we need finish your design review before code implementation, please have a look: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/951

**Comment by isra-fel** [comment]
Since the design review is ongoing let me make this a draft PR. Please mark it as "Ready for review" when you feel comfortable. Thanks

**Comment by dabenhamMic** [comment]
> Since the design review is still ongoing let me make this a draft PR, for repo maintenance purposes. Please mark it as "Ready for review" when you feel comfortable. Thanks

I have updated the open issue: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/951 
added a summary of all new commands and changes

**Comment by dabenhamMic** [comment]
Hey @isra-fel  , I have gone over the errors in the build - there are some stuff that I have fixed (wrong types and other issues that I have resolved) how can we disable some of the errors? please ping me in Teams (My Microsoft alias is: dabenham)

**Comment by dabenhamMic** [comment]
**Remaining errors:**
  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand","Get-AzOperationalInsightsCluster","0","1050","The parameter set '**__AllParameterSets**' for cmdlet 'Get-AzOperationalInsightsCluster' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Get-AzOperationalInsightsCluster'."
    **=> false positive**
  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.SetAzureOperationalInsightsStorageInsightCommand","Set-AzOperationalInsightsStorageInsight","0","1050","The parameter set 'ByWorkspaceObject' for cmdlet 'Set-AzOperationalInsightsStorageInsight' has been removed.","Add parameter set 'ByWorkspaceObject' back to cmdlet 'Set-AzOperationalInsightsStorageInsight'."
    **=> nothing have changed - ByWorkspaceObject still in use in this command**
  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.SetAzureOperationalInsightsStorageInsightCommand","Set-AzOperationalInsightsStorageInsight","0","1050","The parameter set 'ByWorkspaceName' for cmdlet 'Set-AzOperationalInsightsStorageInsight' has been removed.","Add parameter set 'ByWorkspaceName' back to cmdlet 'Set-AzOperationalInsightsStorageInsight'."
    **=> nothing have changed - ByWorkspaceName still in use in this command**
  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.SetAzureOperationalInsightsStorageInsightCommand","Set-AzOperationalInsightsStorageInsight","0","1050","The parameter set '**__AllParameterSets**' for cmdlet 'Set-AzOperationalInsightsStorageInsight' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Set-AzOperationalInsightsStorageInsight'."
    **=> false positive**
  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","1020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSLinkedService'.","Make cmdlet 'Update-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSLinkedService'."
    **=>this was a mistake - now fixed - false alarm**

  "Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","1050","The parameter set '**__AllParameterSets**' for cmdlet 'Update-AzOperationalInsightsCluster' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Update-AzOperationalInsightsCluster'."
    **=> false positive**

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.LinkedServices.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.LinkedServices.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.LinkedStorageAccount.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.LinkedStorageAccount.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Operations.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Operations.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Purge.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Purge.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Tables.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Tables.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Workspaces.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Workspaces.cs:164]: how can parameters be null?

File: src/OperationalInsights/OperationalInsights/DataExports/GetAzureOperationalInsightsDataExportCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/DataExports/GetAzureOperationalInsightsDataExportCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/DataExports/NewAzureOperationalInsightsDataExportCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/DataExports/NewAzureOperationalInsightsDataExportCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/DataExports/RemoveAzureOperationalInsightsDataExportCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/DataExports/RemoveAzureOperationalInsightsDataExportCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/DataExports/UpdateAzureOperationalInsightsDataExportCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/DataExports/UpdateAzureOperationalInsightsDataExportCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/CreatePSDataExportParameters.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/CreatePSDataExportParameters.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSAvailableServiceTiers.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSAvailableServiceTiers.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSClusterSku.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSClusterSku.cs:58]:
```suggestion
                throw new PSArgumentException("SkuCapacity need to be more than1000 GB ");
```

File: src/OperationalInsights/OperationalInsights/Models/PSDataExport.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSDataExport.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSOperation.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSOperation.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSOperationStatus.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSOperationStatus.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSSavedSearchParameters.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSSavedSearchParameters.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSStorageInsight.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSStorageInsight.cs:40]: why is this if block needed?

File: src/OperationalInsights/OperationalInsights/Models/PSTable.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSTable.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSWorkspaceFeatures.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSWorkspaceFeatures.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/PSWorkspaceSku.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/PSWorkspaceSku.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/UpdatePSTableParameters.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/UpdatePSTableParameters.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeBodyFilters.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeBodyFilters.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeResponse.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeResponse.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeStatusResponse.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Models/WorkspacePurge/PSWorkspacePurgeStatusResponse.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/OperationStatus/GetAzureOperationalInsightsOperationStatusCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/OperationStatus/GetAzureOperationalInsightsOperationStatusCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/OperationalInsights.csproj
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/OperationalInsights.csproj:9]: any reason removing this section?

File: src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs:8]:
```suggestion
    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsOperation"), OutputType(typeof(IList<PSOperation>))]
```

File: src/OperationalInsights/OperationalInsights/Tables/GetAzureOperationalInsightsTableCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Tables/GetAzureOperationalInsightsTableCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Tables/SetAzureOperationalInsightsTableCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Tables/SetAzureOperationalInsightsTableCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/WorkspacePurge/GetAzureOperationalInsightsWorkspacePurgeStatusCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/WorkspacePurge/GetAzureOperationalInsightsWorkspacePurgeStatusCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/WorkspacePurge/NewAzureOperationalInsightsPurgeWorkspaceCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/WorkspacePurge/NewAzureOperationalInsightsPurgeWorkspaceCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/Workspaces/UpdateAzureOperationalInsightsWorkspaceSharedKeysCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/Workspaces/UpdateAzureOperationalInsightsWorkspaceSharedKeysCommand.cs:1]: please add license

File: src/OperationalInsights/OperationalInsights/AvailableServiceTiers/GetAzureOperationalInsightsAvailableServiceTiersCommand.cs
Comment (VeryEarly) [inline src/OperationalInsights/OperationalInsights/AvailableServiceTiers/GetAzureOperationalInsightsAvailableServiceTiersCommand.cs:1]: please add license

**Review by VeryEarly — CHANGES_REQUESTED** [review]

**Comment by VeryEarly** [comment]
test cases need to be re-recorded after API upgrade.

File: src/OperationalInsights/OperationalInsights/Az.OperationalInsights.psd1
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Az.OperationalInsights.psd1:15]: change undone

File: src/OperationalInsights/OperationalInsights/Az.OperationalInsights.psd1
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Az.OperationalInsights.psd1:93]: removed

File: src/OperationalInsights/OperationalInsights/AvailableServiceTiers/GetAzureOperationalInsightsAvailableServiceTiersCommand.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/AvailableServiceTiers/GetAzureOperationalInsightsAvailableServiceTiersCommand.cs:1]: what do you mean here?

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.AvailableServiceTiers.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.AvailableServiceTiers.cs:1]: what do you mean? what is the license?

Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.AvailableServiceTiers.cs:1]: added

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Operations.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Operations.cs:1]: added

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.DataExports.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.DataExports.cs:1]: added

File: src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Tables.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Client/OperationalInsightsClient.Tables.cs:1]: added

**Comment by dabenhamMic** [comment]
adding @yoramsinger  - Yabo asked us to re-recorded the api calls.
this will take some more time (I haven't done this before) and will hold us back, please see if you can ask for an exception for us?

File: src/OperationalInsights/OperationalInsights/Models/PSStorageInsight.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Models/PSStorageInsight.cs:40]: removed

**Comment by dabenhamMic** [comment]
@VeryEarly  can we proceed with this pr?

**Comment by wyunchi-ms** [comment]
This PR will introduce breaking changes. Please keep it open until the major release.

File: src/OperationalInsights/OperationalInsights/ChangeLog.md
Comment (dingmeng-xue) [inline src/OperationalInsights/OperationalInsights/ChangeLog.md:67]:
```suggestion
## Version 2.3.1
```

File: src/OperationalInsights/generated/utils/Unprotect-SecureString.ps1
Comment (wyunchi-ms) [inline src/OperationalInsights/generated/utils/Unprotect-SecureString.ps1:13]: Please remove the files under `generated` folder.

**Comment by wyunchi-ms** [comment]
You can skip the api check in the other modules.

**Review by dabenhamMic — COMMENTED** [review]
already fixed

**Review by wyunchi-ms — DISMISSED** [review]

**Comment by wyunchi-ms** [comment]
LGTM

**Review by wyunchi-ms — APPROVED** [review]

File: src/OperationalInsights/OperationalInsights/LinkedStorageAccount/SetAzureOperationalInsightsLinkedStorageAccountCommand.cs
Comment (isra-fel) [inline src/OperationalInsights/OperationalInsights/LinkedStorageAccount/SetAzureOperationalInsightsLinkedStorageAccountCommand.cs:43]: I don't know how frequently the values of DataSourceType change, but if you don't want to block customers to use your latest type, you can change `ValidateSet` to `ArgumentCompleter` which provides the same tab-completion feature but does not validate the argument.

File: src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs
Comment (isra-fel) [inline src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs:20]:
```suggestion
    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "OperationalInsightsOperation"), OutputType(typeof(PSOperation))]
```
The cmdlet does not return a list, instead, it returns the operation for many times, so the output type should be operation

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hello Danial @dabenhamMic , please see my inline comments.
Besides, since a bunch of breaking changes were introduced, could you help us fill in a table like this so we can provide our customer the migration guide? (Do this after the PR is merged so that everything is finalized).
Thanks.

|Cmdlet| Breaking Change| Old Usage| New Usage|
|-|-|-|-|
|| | ||

File: src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs:20]: @isra-fel  is there a way to list all cmdlets that have "breaking changes"?

File: src/OperationalInsights/OperationalInsights/LinkedStorageAccount/SetAzureOperationalInsightsLinkedStorageAccountCommand.cs
Comment (dabenhamMic) [reply src/OperationalInsights/OperationalInsights/LinkedStorageAccount/SetAzureOperationalInsightsLinkedStorageAccountCommand.cs:43]: these are ARM resources types, it can be left as is

**Comment by dabenhamMic** [comment]
Hi @isra-fel  , I have addressed all issues mentioned in your last review

File: src/OperationalInsights/OperationalInsights/LinkedStorageAccount/NewAzureOperationalInsightsLinkedStorageAccountCommand.cs
Comment (isra-fel) [inline src/OperationalInsights/OperationalInsights/LinkedStorageAccount/NewAzureOperationalInsightsLinkedStorageAccountCommand.cs:45]:
```suggestion
```
Since renaming this parameter to plural might not be a good idea, we might just remove this line completely, as there is not going to be breaking change.

File: src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs
Comment (isra-fel) [reply src/OperationalInsights/OperationalInsights/Operations/GetAzureOperationalInsightsOperationCommand.cs]: We have limited types of breaking change. Do a full-text search of [these attributes](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-definition.md#cmdlets) you should be able to list all of them

**Review by isra-fel — CHANGES_REQUESTED** [review]
Others look good to me :)

**Comment by dabenhamMic** [comment]
Hey @wyunchi-ms , @isra-fel 
could you approve and merge this pr?

**Review by isra-fel — APPROVED** [review]
LGTM

**Comment by isra-fel** [comment]
@dabenhamMic Hey Daniel, the PR is good now and I'll merge it once security-tools check finishes. As I said, there's one more thing we need for the release, which is the migration guide. Would you kindly fill in a table like this and  send it to us?

Thanks.

Cmdlet | Breaking Change | Old Usage | New Usage
-- | -- | -- | --

**Comment by dabenhamMic** [comment]
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/dabenham/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/dabenham/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Cmdlet | Breaking Change | Old Usage | New Usage
-- | -- | -- | --
Get-AzOperationalInsightsCluster | The parameter set '__AllParameterSets' for cmdlet   'Get-AzOperationalInsightsCluster' has been removed | no default parameter set | default parameter set is now "list", when providing RG name -   return all clusters for the given RG. Added support for input cluster   resourceID
Update-AzOperationalInsightsCluster | The cmdlet 'Update-AzOperationalInsightsCluster' no longer has output   type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSLinkedService | no cahnge - bug fix |  
Update-AzOperationalInsightsCluster | __AllParameterSets' for cmdlet 'Update-AzOperationalInsightsCluster' has   been removed. |   | now supports ResourceID (instead of manually entering RG name and cluster   name parameters), added support for input cluster object for update



</body>

</html>

**Comment by isra-fel** [comment]
Great, thanks Daniel!

**Pull request merged**
